testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/federation/kerberos/AbstractKerberosSingleRealmTest.java

```diff
@@ -131,35 +139,25 @@ public void usernamePasswordLoginTest() throws Exception {
         // Change editMode to READ_ONLY
         updateProviderEditMode(UserStorageProvider.EditMode.READ_ONLY);
 
-        // Login with username/password from kerberos
-        changePasswordPage.open();
-        loginPage.assertCurrent();
-        loginPage.login("jduke", "theduke");
-        changePasswordPage.assertCurrent();
+        TestAppHelper testAppHelper = new TestAppHelper(oauth, loginPage, appPage);
 
-        // Bad existing password
-        changePasswordPage.changePassword("theduke-invalid", "newPass", "newPass");
-        Assert.assertTrue(driver.getPageSource().contains("Invalid existing password."));
+        testAppHelper.login("jduke", "theduke");
```

Comment:
```suggestion
        Assert.assertTrue(testAppHelper.login("jduke", "theduke"));
```
Would be good to verify the state. I'm aware the intention for that is to simplify the process and mainly execute the methods only as certain "procedures", but in this case it's ok(IMHO).

Reply:
Should be sorted now